src/Fhir.Anonymizer.Core/Utility/DateTimeUtility.cs

```diff
@@ -155,12 +155,15 @@ private static bool IndicateAgeOverThreshold(GroupCollection groups)
             return age > s_ageThreshold; 
         }
 
-        private static int GetDateShiftValue(ElementNode node, string dateShiftKey)
+        private static int GetDateShiftValue(ElementNode node, string dateShiftKey, string dateShiftPrefix)
         {
-            string resourceId = TryGetResourceId(node);
+            if (string.IsNullOrEmpty(dateShiftPrefix))
+            {
+                dateShiftPrefix = TryGetResourceId(node);
```

Comment:
Would we use resource Id as dateshift scope by default?

Reply:
Yes, the default date shift scope is resource. It uses resource Id as dateShiftPrefix to compute shifting amount.